<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectPage</class>
 <widget class="QWidget" name="ProjectPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 76, 76);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>541</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Please select which analysis tools you would like to use</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="GCC">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>GCC -fanalyzer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Flawfinder">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Flawfinder</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CPP">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>CPPcheck</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AnalyizeBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 204, 150);</string>
   </property>
   <property name="text">
    <string>Analyze</string>
   </property>
  </widget>
  <widget class="QLabel" name="analysisLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Click to begin analysis</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="ReportspageBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 204, 150);</string>
   </property>
   <property name="text">
    <string>Go to Reports Page</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
